packages/components/src/breadcrumbs/index.tsx

```diff
@@ -0,0 +1,146 @@
+import {
+	__experimentalText as Text,
+	__experimentalHStack as HStack,
+	Button,
+	VisuallyHidden,
+} from '@wordpress/components';
+import { useResizeObserver, useMergeRefs } from '@wordpress/compose';
+import { __ } from '@wordpress/i18n';
+import { useState, forwardRef, useRef } from 'react';
+import Menu from '../menu';
+import { BreadcrumbProps, BreadcrumbItemProps } from './types';
+import './style.scss';
+
+function BreadcrumbsMenu( { items }: { items: BreadcrumbItemProps[] } ) {
+	return (
+		<span className="breadcrumb__item-wrapper">
+			<Menu placement="bottom-start">
+				<Menu.TriggerButton
+					render={
+						<Button
+							size="compact"
+							className="breadcrumb__item"
+							text="…"
+							label={ __( 'More breadcrumb items' ) }
+						/>
+					}
+				/>
+				<Menu.Popover>
+					{ items.map( ( item, index ) => (
+						<Menu.Item
+							key={ `${ item.label }-${ index }` }
+							render={
+								<a href={ item.href } onClick={ item.onClick } className="breadcrumb__item" />
+							}
+						>
+							<Menu.ItemLabel>{ item.label }</Menu.ItemLabel>
+						</Menu.Item>
+					) ) }
+				</Menu.Popover>
+			</Menu>
+		</span>
+	);
+}
+
+function BreadcrumbItem( { item: { label, href, onClick } }: { item: BreadcrumbItemProps } ) {
+	return (
+		<span className="breadcrumb__item-wrapper">
+			<a href={ href } onClick={ onClick } className="breadcrumb__item">
+				{ label }
+			</a>
+		</span>
+	);
+}
+
+function UnforwardedBreadcrumbs(
+	{ items, showCurrentItem = false, isCompact = false }: BreadcrumbProps,
```

Comment:
Not a blocker, but for better extensibility, we usually accept and forward all HTML props/attributes for the underlying element

```suggestion
	{ items, showCurrentItem = false, isCompact = false, ...restProps }: BreadcrumbProps & React.HTMLAttributes< HTMLNavElement >,
```

Reply:
It seems there is no `HTMLNavElement`. From my quick search I didn't find the alternative. I'll look into it tomorrow more, but if you know it, please share :)